ChessBoard.Api/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3"
+services:
+  mongo:
+    image: mongo
+    container_name: mongo
+    ports:
+      - "27017:27017"
+    volumes:
+      - mongodbdata:/data/db
+    networks:
+      - chessboard-network
+
+  chessboard:
```

Comment:
What happens if mongo fails to launch or chessboard starts before mongo? (usual case with mmsql)
Please consider adding "depends_on" 
and adding a restart policy https://blog.knoldus.com/restart-policies-in-a-docker-container/

Reply:
```
version: "3"
services:
  mongo:
    image: mongo
    container_name: mongo
    ports:
      - "27017:27017"
    volumes:
      - mongodbdata:/data/db
    networks:
      - chessboard-network
    restart: unless-stopped  

  chessboard:
    build:
      context: .
      dockerfile: Dockerfile
    image: chessboardapi:v1.0
    container_name: chessboardapi
    ports:
      - "8080:80"
    environment:
      - MongoDbSettings:Host=mongo
    depends_on:
      - mongo  
    networks:
      - chessboard-network
    restart: unless-stopped

networks:
  chessboard-network:

volumes:
  mongodbdata:
```

Maybe something like this?

Reply:
yes, exactly :)